templates/certification/component-details.html

```diff
@@ -1,56 +1,99 @@
-        <td>{{ component.part_number }}</td>
-        </tr>
-        <tr>
-        <th scope="row">Device Category</th>
-        <td>{{ component.category }}</td>
-        </tr>
-        <tr>
-        <th scope="row">Note</th>
-        <td>{{ component.note }}</td>
-        </tr>
-    </tbody>
-</table>
+{% block content %}
+<section class="p-strip--suru-topped">
+  <div class="row u-sv3">
+    <div class="col-12">
+      <h1>{{ component.vendor_name }} {{ component.model }}</h1>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <h2 class="u-sv2">Components certified in Ubuntu releases</h2>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <table class="u-sv1" aria-label="Components certified in ubuntu releases">
+        <thead>
+          <tr>
+            <th>Release</th>
+            <th>Status</th>
+            <th>Third-party driver</th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for name, details in component.lts_releases.items() %}
+          <tr>
+            <td>{{ details.0.release }}</td>
+            <td>{{ details.0.status.title() }}</td>
+            <td>{% if details.0.third_party_driver != True %}No{% else %}Yes{% endif %}</td>
+          </tr>
+          {% endfor %}
+        </tbody>
+      </table>
+    </div>
+  </div>
+  <div class="row u-sv1">
+    <div class="col-12">
+      <h2 class="u-sv2">Component details</h2>
+      <table class="p-table--component-details" aria-label="Component details table">
+        <tbody>
+          <tr>
+            <th class="p-muted-text">Vendor</th>
+            <td>{{ component.vendor_name }}</td>
+          </tr>
+          <tr>
+            <th class="p-muted-text">Model name</th>
+            <td>{{ component.model }}</td>
+          </tr>
+          <tr>
+            <th class="p-muted-text">Identifier</th>
+            <td>{{ component.identifier }}</td>
+          </tr>
+          <tr>
+            <th class="p-muted-text">Subsystem Identifier</th>
+            <td>{{ component.subsystem_identifier}}</td>
+          </tr>
+          <tr>
+            <th class="p-muted-text">Hardware Vendor make</th>
+            <td>{{ component.hardware_vendor_make }}</td>
+          </tr>
+          <tr>
+            <th class="p-muted-text">Vendor make</th>
+            <td>{{ component.vendor_make }}</td>
+          </tr>
+          <tr>
+            <th class="p-muted-text">Part number</th>
+            <td>{{ component.part_number }}</td>
+          </tr>
+          <tr>
+            <th class="p-muted-text">Device category</th>
+            <td>{{ component.category }}</td>
+          </tr>
+          <tr>
+            <th class="p-muted-text">Note</th>
+            <td>{{ component.note }}</td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <h2>Present in {{ machines|length }} certified system{% if machines|length > 1%}s{% endif %}</h2>
```

Comment:
Space between the 1 and %? 
It was weird initially I thought you were doing the condition against a percentage number, I thought how can this work?